PWGGA/GammaConv/AliV0ReaderV1.h

```diff
@@ -131,6 +131,7 @@ class AliV0ReaderV1 : public AliAnalysisTaskSE {
     Int_t              GetNumberOfPrimaryTracks()                       {return fNumberOfPrimaryTracks;}
     Double_t           GetSphericity()                                  {return fSphericity;}
     void               SetCalcSphericity(Bool_t set)                    {fCalcSphericity=set; return;}
+    void               SetCalcSector(Bool_t set)                        {fCalcSector=set; return;}
```

Comment:
don't you need a GetCalcSector as well?

Reply:
and how do you access fPtMaxSector afterwards, I don't get it

Reply:
I am guessing fPtMaxSector is a private member of fVoareder, but its hard to understand seeing only bits and pieces of the code